nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberDetail.js

```diff
@@ -147,6 +148,10 @@ export default function SubscriberDetail(props: {
         </Grid>
       </AppBar>
       <Switch>
+        <Route
+          path={relativePath('/config/json')}
```

Comment:
is this per subscriber or for all subscribers ?

Reply:
Per subscriber

Reply:
if this per subscriber, shouldn't we have a subscriber id in the path ?

Reply:
It's already in the path

Reply:
We add it here: https://github.com/magma/magma/blob/master/nms/app/fbcnms-projects/magmalte/app/views/subscriber/SubscriberOverview.js#L127